handler/src/main/java/io/netty/handler/ssl/ReferenceCountedOpenSslEngine.java

```diff
@@ -1090,16 +1087,14 @@ public final SSLEngineResult unwrap(
             final ByteBuffer[] dsts, int dstsOffset, final int dstsLength) throws SSLException {
 
         // Throw required runtime exceptions
-        ObjectUtil.checkNotNull(srcs, "srcs");
+        checkNotNullWithIAE(srcs, "srcs");
```

Comment:
While this is different to what happened before (before it was a NPE) I think the new behaviour is more correct

Reply:
I should have commented this one. I compared it to the other checks and switched it therefor.

Reply:
no problem... like I said I think it is now more correct :)